client/app/bundles/shared/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,48 @@
+@import "../base.scss";
+
+body {
+  font-family: $font-family;
+}
+
+.fill select{
+  background:  #6D0839 url('./down.svg') no-repeat right;
+
+  @include selectDropdown(
+    $padding: 13px 32px,
+    $font-size: $font18
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.ghost select{
+  background:  #9a9999 url('./down.svg') no-repeat right;
+
+  @include selectDropdown(
+    $padding: 17px 35px,
+    $font-size: $font22
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.ghostsmall select {
+  background:  #9a9999 url('./down.svg') no-repeat right;
+
+  @include selectDropdown(
+    $padding: 13px 32px,
+    $font-size: $font18
+  );
+  &:focus {
+    outline: none;
+  }
+}
+
+.select_dropdown select {
+  padding-left: 10px;
+  cursor: pointer;
+  display: inline-block;
```

Comment:
Let's make sure it's the container `.select_dropdown` that gets the `inline-block` and not the `select`!